### PR TITLE
Use Rails 4.2.0 specifically for web app compatibility

### PR DESCRIPTION
**Pull request by nmcclay**

### CODE REVIEW

File: forever_style_guide.gemspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails", "~> 4.2.1"
+  s.add_dependency "rails", "=4.2.0"
```
Comment (csto) [inline forever_style_guide.gemspec]: Shouldn't this be "~>4.2.0"?

Comment (nmcclay) [inline forever_style_guide.gemspec]: I actually tried that first, but for whatever reason with that when I did bundle install it was still running 4.2.1 for me.

Comment (lorily16) [inline forever_style_guide.gemspec]: @csto It looks like I have to use '=' for web app as well. For some reason, bundle install still install the latest version for me if I use "~> 4.2.0". Do you know how to work around it?

**Comment by lorily16** [comment]
LGTM!

**Pull request merged**